app/models/miq_expression/field.rb

```diff
@@ -53,8 +63,58 @@ def does_not_match(other)
     arel_attribute.does_not_match(other, escape, case_sensitive)
   end
 
+  def contains(other)
+    raise unless associations.one?
+    reflection = reflections.first
+    arel = eq(other)
+    arel = arel.and(Arel::Nodes::SqlLiteral.new(extract_where_values(reflection.klass, reflection.scope))) if reflection.scope
+    model.arel_attribute(:id).in(
+      target.arel_table.where(arel).project(target.arel_table[reflection.foreign_key]).distinct
+    )
+  end
+
   private
 
+  class WhereExtractionVisitor < Arel::Visitors::PostgreSQL
```

Comment:
@NickLaMuro what does `< Arel::Visitors::PostgreSQL` do?

If this were extending `to_sql`, then I can see how this would be injected into the visitor pattern. But I'm unsure what it does here. (I know you just copied the code. but am curious if you had insight)